CONTRIBUTING.rst

```diff
@@ -448,6 +448,50 @@ fetch and checkout a remote branch like this::
     https://notes.envato.com/developers/rebasing-merge-commits-in-git/
 .. _`Rebase`: https://help.github.com/rebase/
 
+.. _contributing-docker-development:
+
+Developing and Testing with Docker
+----------------------------------
+
+Because of the many components of celery, such as a broker and backend,
+`Docker`_ and `docker-compose`_ can be utilized to greatly simplify the
+development and testing cycle.
+
+The docker components can be found within the :file:`docker/` folder and the
+docker image can be built via:
+
+.. code-block:: console
+
+    $ docker-compose build celery
+
+and run via:
+
+.. code-block:: console
+
+    $ docker-compose run --rm celery <command>
+
+where <command> is a command to execute in a docker container. The `--rm` flag
+indicates that the container should be removed after it is exited and is useful
+to prevent accumulation of unwanted containers.
+
+Some useful commands to run:
+
+* ``bash``
+
+    To enter the docker container like a normal shell
+
+* ``make test``
```

Comment:
I think this needs to be replaced with the proper `tox` command?

Reply:
I can add that to the section. `tox` takes a lot longer to run so I think it's useful to provide the simple `make test` command.